cmd/bblfsh-tools/common.go

```diff
@@ -0,0 +1,73 @@
+package main
+
+import (
+	"context"
+	"io/ioutil"
+
+	"google.golang.org/grpc"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/bblfsh/sdk/protocol"
+	"github.com/bblfsh/sdk/uast"
+	"github.com/bblfsh/tools/tools"
+	errors "srcd.works/go-errors.v0"
+)
+
+var (
+	ErrorInvalidTool = errors.NewKind("invalid tool %s")
```

Comment:
The usual convention (which we follow) is prefixing these with just `Err` instead of `Error`.